Nanoleaf-wpf/Winleafs.Wpf.csproj

```diff
@@ -43,6 +43,9 @@
     <Reference Include="Hardcodet.Wpf.TaskbarNotification, Version=1.0.5.0, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>packages\Hardcodet.NotifyIcon.Wpf.1.0.8\lib\net451\Hardcodet.Wpf.TaskbarNotification.dll</HintPath>
     </Reference>
+    <Reference Include="JsonMigrator, Version=1.0.1.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
I don't believe this project needs reference to JsonMigrator

Reply:
For my instance, it actually does and will crash without it.
I can do more google work on this but this is a more common issue for me.

Reply:
Please do since this will occur in the future again with some other package if the problem is on your side

Reply:
```
System.IO.FileNotFoundException: 'Could not load file or assembly 'JsonMigrator, Version=1.0.1.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. The system cannot find the file specified.'
```

When starting the app without this dependency. More research later

Reply:
Opened issue 47 for this problem: https://github.com/StijnOostdam/Winleafs/issues/47